common.blocks/dropdown/dropdown.bemhtml

```diff
@@ -1,4 +1,8 @@
 block('dropdown')(
+    def()(function() {
+        applyNext({ _hasTick : this.mods['has-tick'] });
```

Comment:
You can check `this.mods['has-tick']` in any template, so you shouldn't create redundant variable `_hasTick`.